src/Pos/Explorer/DB.hs

```diff
@@ -28,6 +30,17 @@ import           Pos.Txp.Toil          (Utxo)
 import           Pos.Util.Chrono       (NewestFirst (..))
 
 ----------------------------------------------------------------------------
+-- Types
+----------------------------------------------------------------------------
+
+type Page = Integer
```

Comment:
It's a simpler construct introduced in `explorer`, a `Page` contains 10 `Block`s - https://github.com/input-output-hk/cardano-sl-explorer/blob/master/src/Pos/Explorer/Web/Api.hs#L48-L60
This simplified the `explorer` socket.io implementation quite a bit, as well as made it much simpler from the frontend perspective.